scripts/fix_centos8.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
```

Comment:
Add `set -e` otherwise some commands here can fail.

The original code of 
```
RUN if grep -q "CentOS Linux 8" /etc/os-release ; then \
        sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*; \
        sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*; \
        yum install -y dnf-plugins-core; \
        dnf config-manager --set-enabled powertools; \
        yum clean all; \
```
was broken as well, because it didn't use `&&` or `set -e`, and that's buggy.

Always use `set -e` in your scrpts :)